include/rocksdb/env.h

```diff
@@ -364,6 +364,8 @@ class Env {
   // Priority for requesting bytes in rate limiter scheduler
   enum IOPriority { IO_LOW = 0, IO_HIGH = 1, IO_TOTAL = 2 };
 
+  enum IOType { IO_UNCATEGORIZED = 0, IO_FLUSH = 1, IO_COMPACTION = 2 };
```

Comment:
I want to avoid adding `IOType` to rocksdb code. I think the rust `FileSystemInspector` can obtain the thread-local IO type by itself.